capability/capability_test.go

```diff
@@ -48,17 +62,7 @@ func TestLastCap(t *testing.T) {
 
 func TestListSupported(t *testing.T) {
 	list, err := ListSupported()
```

Comment:
Looks like `list` here is shadowing `list()` in enum_gen.go; perhaps `supported` would work

Reply:
This is not part of this PR, this code is from #153.

We can rename this package capability_test :) to make this shadowing go away 🕺🏻

Reply:
Code moved to #162; in there I actually followed my own idea of renaming the test package.